Amazon.QLDB.Driver/driver/BaseQldbDriver.cs

```diff
@@ -0,0 +1,60 @@
+﻿/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: please remove the year. Recently, the guidance changed to not include the years anymore. doesn't need to be addressed in this pr.